ncm-filesystems/src/main/perl/filesystems.pod

```diff
@@ -16,6 +16,8 @@ from C<< /etc/fstab >>, using ncm-fstab.
 The component doesn't provide any special resources at the moment. It
 just watches for changes on C<< /system/filesystems >> and C<< /system/blockdevices >>
 and creates new filesystems, if needed.
+You can also use ncm-filesystems to replace ncm-fstab :
+If C<< manage_blockdevs >> is set to false, only the ncm-fstab code will run.
```

Comment:
It is not a lot of code, but probably most useful for testing: With the flag false, I can use tests of ncm-fstab to check ncm-filesystems does at least the same. But I can remove it quite simple :smile: